contrib/asmap/asmap-tool.py

```diff
@@ -6,7 +6,9 @@
 import argparse
 import sys
```

Comment:
nit: sys isn't in the right alphabetical order here. You could reorder it while you are at it.

Reply:
Moving `sys` doesn't fully fix the import block because it is separated from the next function definition by only one newline. If I added another newline below the import block fix it, I should probably also add additional newlines between function definitions for consistency's sake. Not sure if any/how much refactoring should be included here. wdyt?